tabular/src/autogluon/tabular/learner/abstract_learner.py

```diff
@@ -147,11 +147,23 @@ def predict_proba(self, X: DataFrame, model=None, as_pandas=True, as_multiclass=
                                                         inverse_transform=inverse_transform)
         return y_pred_proba
 
-    def predict(self, X: DataFrame, model=None, as_pandas=True, inverse_transform=True, transform_features=True):
+    def predict(self,
+                X: DataFrame,
+                model=None,
+                as_pandas=True,
+                inverse_transform=True,
+                transform_features=True,
+                *,
+                decision_threshold: float = None,
```

Comment:
nit: `decision_threshold: Optional[float] = None`

Also, add types for other args above

Reply:
We can add type hints in learner as part of a separate PR